### PR TITLE
Adding Contributors key to metadata schema

### PR DESCRIPTION
**Pull request by damienjburks**

### CODE REVIEW

**Comment by eddie-knight** [comment]
@damienjburks 

This is blocked by [CODEOWNERS#L48](https://github.com/finos/common-cloud-controls/blob/main/.github/CODEOWNERS#L48) (codeowners file doesn't hoist)

File: services/storage/object/metadata.yaml
Comment (eddie-knight) [inline services/storage/object/metadata.yaml:28]: seeing this layed out, I like this more than my previous suggestion :+1: LGTM.

**Comment by damienjburks** [comment]
@eddie-knight raising a PR to fix that CodeOwners file. give me a min.

**Comment by sshiells-scottlogic** [comment]
@damienjburks  can you add an example to the templates file? https://github.com/finos/common-cloud-controls/blob/main/docs/governance/community-guidelines/templates/metadata.yaml

**Comment by damienjburks** [comment]
@sshiells-scottlogic i've updated the yaml file - can you approve please?

**Review by eddie-knight — APPROVED** [review]

**Pull request merged**
